docs/user-guide/masking.ipynb

```diff
@@ -91,13 +105,10 @@
    "outputs": [],
    "source": [
     "a = sc.DataArray(\n",
-    "    data = sc.Variable(dims=['y', 'x'], values=np.arange(1.0, 7.0).reshape((2, 3))),\n",
-    "    coords={\n",
-    "        'y': sc.Variable(dims=['y'], values=np.arange(2.0), unit=sc.units.m),\n",
-    "        'x': sc.Variable(dims=['x'], values=np.arange(3.0), unit=sc.units.m)},\n",
-    "    masks={\n",
-    "        'x': sc.Variable(dims=['x'], values=[False, False, True])}\n",
-    "    )\n",
+    "    data=sc.array(dims=['y', 'x'], values=np.arange(1.0, 7.0).reshape((2, 3))),\n",
```

Comment:
Use `sc.arange` and `fold` instead?

Reply:
I felt that was more confusing since it requires a named dummy dimension.